### ISSUE TITLE
How to configure Azure with monorepo? 

### ISSUE THREAD

**Issue by ruslanguns** [TrilonIO/nestjs-azure-functions#1]
Thanks for contributing with this sample and works amazing with a nestjs application, but if I have a monorepo with Library or Apps behing the repo the Nest CLI does not compile the configuration for Azure.

I would be really appreciated if you give me a solution.

By the way, I same happen with Deploy to Zeit Now. Default settings does not work.

Regards!

**Comment by MarkPieszak** [comment]
I haven't experimented yet with deploying the Monorepos in the similar fashion, but I imagine it might just need to know the correct `dist/` location that the files get built to, and it should work just the same! Let me know if you figure it out somehow!

**Comment by ruslanguns** [comment]
@MarkPieszak absolutely. I will let you know. Thanks for responding. I will try again in this week .

**Comment by drieshooghe** [comment]
@ruslanguns Did you get this to work?

**Comment by dinohorvat** [comment]
@ruslanguns I am facing the same issue. Have you managed to resolve it? No matter how I configure it, the `nest build` will not generate the `main` folder and its files (to `dist`). It seems like azure configuration gets ignored while building the libraries / projects.

**Comment by ruslanguns** [comment]
@dinohorvat I need to retake this, I will try to reproduce this issue again and let you know.

**Comment by akirashimosoeda** [comment]
As a workaround, you can move `${root}/main/index.ts` in `${app_root_path}/src/main/index.ts`. This way, your file will be compiled to JS in the `dist` folder. Update `scriptFile` prop in `functions.json` accordingly.
